deluge/common.py

```diff
@@ -720,6 +720,23 @@ def parse_human_size(size):
     raise InvalidSize(msg % (size, tokens))
 
 
+def wrap_url_with_html(text):
```

Comment:
Hmm I wonder if `anchorify_urls` might be a bit more succinct

Reply:
sounds good to me